cmake/toolchain/gnuarmemb/Kconfig

```diff
@@ -6,3 +6,8 @@
 config TOOLCHAIN_GNUARMEMB
 	def_bool y
 	select HAS_NEWLIB_LIBC_NANO
+
+config NEWLIB_SUPPORTED
```

Comment:
We already have `TOOLCHAIN_HAS_NEWLIB` CMake variable that gets passed down to Kconfig as `CONFIG_TOOLCHAIN_HAS_NEWLIB`.

It should not be necessary to introduce `NEWLIB_SUPPORTED`.

Reply:
Ah, I had no idea that was available as a config variable. Thanks!